index.bs

```diff
@@ -3380,10 +3382,12 @@ partial dictionary MLOpSupportLimits {
   MLBinarySupportLimits logicalAnd;
   MLBinarySupportLimits logicalOr;
   MLBinarySupportLimits logicalXor;
+  MLBinarySupportLimits isNaN;
```

Comment:
Use `MLLogicalNotSupportLimits`? Or you may want to rename to something general.

Reply:
Doh yes, like unary logicalNot. `MLLogicalNotSupportLimits` makes sense to me since it does a test and returns a Boolean value, very similar to the others.